variables.tf

```diff
@@ -40,6 +41,12 @@ variable "lacework_integration_name" {
   description = "The name of the integration in Lacework"
 }
 
+variable "lacework_audit_policy_name" {
+  type    = string
+  default = ""
+  description = "The name of the custom audit policy (which extends SecurityAudit) to allow Lacework to read configs.  Defaults to lwaudit-policy-$${random_id.uniq.hex} when empty"
```

Comment:
Not sure you can do that. 🤨

Reply:
This is done via `locals` in `main.tf`.  This description is `$$` escaped string literal.